app/lib/core/view_models/account_screen_view_model.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:app/core/models/transporter.dart';
+import 'package:app/core/services/authentication/auth_service.dart';
+import 'package:app/core/services/database/database_service.dart';
+import 'package:app/core/services/navigation/nav_service.dart';
+import 'package:app/core/services/service_locator.dart';
+import 'package:app/ui/views/account/account_editing_screen.dart';
+import 'base_view_model.dart';
+
+class AccountScreenViewModel extends BaseViewModel {
+  final NavigationService _navigationService = locator<NavigationService>();
+  final DatabaseService _databaseService = locator<DatabaseService>();
+  final AuthenticationService _authenticationService =
+      locator<AuthenticationService>();
+
+  Transporter _transporter;
+  Transporter get transporter => _transporter;
+
+  void loadTransporterInfo() {
```

Comment:
- [ ] This stream subscription, `getUserInRealTime` needs to be disposed of like `TextEditingControllers`

See the HistoryScreenViewModel for example.